controllers/machinepools.go

```diff
@@ -115,7 +115,10 @@ func getOCMClusterID(r *AppWrapperReconciler) error {
 	// Get the client for the resource that manages the collection of clusters:
 	collection := connection.ClustersMgmt().V1().Clusters()
 
-	response, _ := collection.List().Search(fmt.Sprintf("external_id = '%s'", internalClusterID)).Size(1).Page(1).SendContext(ctx)
+	response, err := collection.List().Search(fmt.Sprintf("external_id = '%s'", internalClusterID)).Size(1).Page(1).SendContext(ctx)
+	if err != nil {
+		klog.Errorf(`Error getting cluster id: %v`, err)
+	}
```

Comment:
Great to add some logging here.

Should something similar be added to line 43 as well, when creating machine pools?

https://github.com/project-codeflare/instascale/blob/9098de3aa90d53df3b360853d769512ebcd8a1e9/controllers/machinepools.go#L43